package com.college.attendance.repository;
import com.college.attendance.model.Subject;
import org.springframework.data.jpa.repository.JpaRepository;
public interface SubjectRepository extends JpaRepository<Subject,Long>{}
